hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/OzoneManagerServiceProviderImpl.java

```diff
@@ -350,8 +357,7 @@ public void syncDataFromOM() {
         // Pass on DB update events to tasks that are listening.
         reconTaskController.consumeOMEvents(new OMUpdateEventBatch(
             omdbUpdatesHandler.getEvents()), omMetadataManager);
-      } catch (IOException | InterruptedException | RocksDBException e) {
-        Thread.currentThread().interrupt();
```

Comment:
Shouldn't `interrupt()` be kept for `InterruptedException` and avoided only for the rest?